database/postgres/users.go

```diff
@@ -156,7 +155,7 @@ func (u *userRepo) FindUserByToken(ctx context.Context, token string) (*datastor
 
 func (u *userRepo) FindUserByEmailVerificationToken(ctx context.Context, token string) (*datastore.User, error) {
 	user := &datastore.User{}
-	err := u.db.QueryRowxContext(ctx, fmt.Sprintf("%s AND email_verification_token = $1;", fetchUsers), token).StructScan(user)
+	err := u.db.GetDB().QueryRowxContext(ctx, fmt.Sprintf("%s AND email_verification_token = $1;", fetchUsers), token).StructScan(user)
```

Comment:
```suggestion
	err := u.db.GetReadDB().QueryRowxContext(ctx, fmt.Sprintf("%s AND email_verification_token = $1;", fetchUsers), token).StructScan(user)
```

Reply:
What if the invite token is not replicated on time?

Reply:
Hmm, the record would be replicated before the email is even delivered. But we can route it to the primary.

Reply:
All right